src/routes/api.js

```diff
@@ -80,8 +115,9 @@ apiRouter.get('/cities', getPlacesFromGoogle);
 apiRouter.get('/plans', getAllPlans);
 // apiRouter.get('/plans/:id', getPlanById)
 apiRouter.post('/plans', savePlanInDB);
-apiRouter.post('/places', savePlaceInDB);
 apiRouter.delete('/plans/:id', removePlanInDB);
+// should it be /?plan/places/:id?
```

Comment:
If places is part of plans then potentially? Nothing wrong with it sharing handler methods in the same place with /plans and / places